core/urls.py

```diff
@@ -1,5 +1,6 @@
 from django.urls import path
 from django.contrib.auth.decorators import login_required
+from . import views
```

Comment:
Please don't use relative imports, be specific and import it like this-
```
from core.views import about, contact
```

Reply:
resolve